kratos/python/add_sparse_matrices_to_python.cpp

```diff
@@ -84,10 +82,11 @@ void AddSparseMatricesToPython(pybind11::module& m)
     //************************************************************************************************************
     //************************************************************************************************************
     py::class_<CsrMatrix<double,IndexType>, CsrMatrix<double,IndexType>::Pointer >(m, "CsrMatrix")
+    .def(py::init<>())
```

Comment:
```suggestion
```

I would not add this one as it can lead to compatibility issues with the distributed version

Reply:
i would prefer to leave this as it is, as it allows doing some low level manipulation in order to construct the matrix from arrays (see e.g. what is being done in the amgcl_conversion_utils)

on the other side this cannot be done for the distributed matrix

Reply:
this is to allow some low level operations for the serial matrix (see e.g. what is done in the amgcl_conversion_utils)

Reply:
ok up to you